### ISSUE TITLE
July 12th 2024 the issue is fixed

### ISSUE THREAD

**Issue by Onyxmia666** [yuliskov/SmartTube#3683]
For those of you who uninstalled smart tube out of frustration if you still have it just wait until you get the update I just got the update for smart to fix that error and now it's fixed I knew if we just waited and we're patient they were going to work on it so the issue has been solved for me I'm back on Smart tube I was using regular YouTube and it was driving me nuts and all those advertisements and pop ups and all this kind of it's just all kinds of advertisements it's annoying so I kept my smart tube where it was I signed on today and I had an update so after I updated it I restarted my TV my fire My Fire TV and it's working back to normal if it's not working for you yet just be patient you'll get the update

**Comment by TaiyakiFox** [comment]
Thanks to the developers to have fixed this problem so fast!

There's still the problem of missing resolutions above 1080p though, I hope this can be fixed too.

**Comment by OshekharO** [comment]
Still not working on fire stick showing 403 on all resolutions.

**Comment by PowerWiesel** [comment]
> Still not working on fire stick showing 403 on all resolutions.

not true

watching 1080p60 on fire stick all day and it works fine

**Comment by OshekharO** [comment]
> > Still not working on fire stick showing 403 on all resolutions.
> 
> not true
> 
> watching 1080p60 on fire stick all day and it works fine

![IMG_20240712_193554](https://github.com/user-attachments/assets/e1a4ef40-5ad5-470e-a232-f085c51968fe)

I don't know about you but it's true for me

**Comment by PowerWiesel** [comment]
> I don't know about you but it's true for me

you made a general statement that it's not working, so I did the same.

Working for me on two 4k first gen fire sticks  - both latest beta and stable. But it looks like just avc and 1080p is available for me right now. Can't test 4k since I don't get the option.

**Comment by GrafVonKrolock** [comment]
Definitely NOT working.
Reinstalling also does nothing. Only way around it for me is to open a Live video first. After that, normal videos will play. Without that, 403 shows up when I first open the app (or when the app wakes from sleep).

**Comment by fullmetalbrackets** [comment]
Having the same issue. 403 error on all videos but does work if I play a live video first. (Thanks for this workaround by the way! Better than it not working at all.) Everything was working perfect again yesterday, didn't install the latest update (the one that says not to install if you have no problems), today got the error, installed update, still get error unless live video is played first.

**Comment by gate1975mlm** [comment]
4K not working again on Shield TV! :(

And I also see the 403 error at random times.

**Comment by kushariforyou** [comment]
Same here, 4K not working on Sony TV.

**Comment by PowerWiesel** [comment]
> Definitely NOT working. Reinstalling also does nothing. Only way around it for me is to open a Live video first. After that, normal videos will play. Without that, 403 shows up when I first open the app (or when the app wakes from sleep).

it definitely IS working. Watching while typing this.

**Comment by kushariforyou** [comment]
@PowerWiesel Now mine is working again, however just because something works for you, doesn't mean it's fixed. If others are having issues, that means the issue is still persistent. If it works for you, then it means that it works for you, doesn't mean that the issue is 100% fixed.

**Comment by PowerWiesel** [comment]
@kushariforyou  I'm 100% with you. But just because it doesn't work for some people, it doesn't mean it doesn't work for anybody else. 

I hate when people do a general "does not work" it the bug report, because that's often not true. I doesn't work for them - that's all.

I see this behavior every day and this time I just had to say something. It makes my blood boil when I see how entitled some of the bug reports in this repo are

**Comment by kushariforyou** [comment]
@PowerWiesel Yup I know what you mean. Weird one, it wasn't working, then when you replied, it worked. Next time it stops working I'll ping you again 🤣

**Comment by PowerWiesel** [comment]
@kushariforyou deal :laughing:

**Comment by GrafVonKrolock** [comment]
> > Definitely NOT working. Reinstalling also does nothing. Only way around it for me is to open a Live video first. After that, normal videos will play. Without that, 403 shows up when I first open the app (or when the app wakes from sleep).
> 
> it definitely IS working. Watching while typing this.

Learn to read.

**Comment by galmok** [comment]
I am using v22.41 and I get 403 errors. This version worked yesterday, but not today.

**Comment by angrykid8** [comment]
Yep not working now for me either on the newest beta

**Comment by escortrs** [comment]
All working ok for the moment, I did have the error at first but stopping the app and clearing the Cache/Data and restarting it works fine.

**Comment by odtgit** [comment]
Works for me at 1080p30fps avc

**Comment by A4Alpha** [comment]
I doubt people who uninstalled smarttube will look in here since they're busy creating new issues. Please close this issue @Onyxmia666 and continue under #3606

**Comment by ACR-Jeff** [comment]
The issue has arrived again, 
Today now non live steam videos aren't playing at all. It brings up the player, Yet clicking play does nothing. I've tried to force close the app, Clearing cache and play again, I've tried to reboot my device, Same thing.
https://github.com/yuliskov/SmartTube/issues/3674

**Comment by ACR-Jeff** [comment]
Few hours later and everything started working again on it's own, Except still getting a 403 error message, Waiting a minute and the videos play.

**Comment by THGhost** [comment]
Definitely not working. I'm getting the 403 message on all videos on my Nvidia Shield Pro and on my friend's Android TV. I've switched to the Stable version for now 👍

**Comment by Gerald-Lofton** [comment]
@Onyxmia666 can you please close this issue to reduce spam of same issue being opened and follow under [#3606 ](https://github.com/yuliskov/SmartTube/issues/3606). Stable update [22.38](https://github.com/yuliskov/SmartTube/releases/tag/22.38s) acknowledged the 403 response status code's and has a _temporary_ resolution workaround that just got pushed out to me at least and videos are working as expected currently. I am on a **Chromecast with Google TV**
